avocado/utils/pmem.py

```diff
@@ -118,6 +124,69 @@ def run_daxctl_list(self, options=''):
         cmd = '%s list %s' % (self.daxctl, options)
         return json.loads(process.system_output(cmd))
 
+    def dax_online_memory(self, device, region='', no_movable=False):
+        """Online memory from a given devdax device
```

Comment:
Is this setting or getting the value? It should be clear on this docstring.

Reply:
`Online` here is a verb here, this function "online"s memory from a given device. Hope it is clear?

Reply:
Since `Online` is an "adjective" or an "adverb" IMO it is still confusing, specially for someone that is not familiar with this concept. I would recommend "set_dax_memory_online()"  or something more explicit.